ott/tools/plot.py

```diff
@@ -107,6 +126,41 @@ def barycenters(arg: Union[transport.Transport, jnp.ndarray],
 
   if isinstance(arg, transport.Transport):
     ot = arg
-    return _barycenters(ax, ot.geom.y, ot.a, ot.b, ot.matrix, **kwargs)
-
-  return _barycenters(ax, arg, a, b, matrix, **kwargs)
+    _barycenters(ax, ot.geom.y, ot.a, ot.b, ot.matrix, **kwargs)
+    return ax
```

Comment:
same, i would rather return the ax in the _barycenters function

Reply:
done.